ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,225 @@
+12.2 "--sync--" sync / 14:[^:]*:Silkie:7759:/ window 15,15
+14.9 "Fizzling Suds" sync / 1[56]:[^:]*:Silkie:7759:/
+21.0 "Soap's Up" sync / 1[56]:[^:]*:Silkie:775A:/
+22.0 "Fizzling Duster" sync / 1[56]:[^:]*:Silkie:775D:/
+32.2 "Dust Bluster" sync / 1[56]:[^:]*:Silkie:776C:/
+38.4 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+43.5 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+50.1 "--sync--" sync / 1[56]:[^:]*:Silkie:(7751|7752):/
+51.6 "Squeaky Clean 1" sync / 1[56]:[^:]*:Silkie:7753:/
+53.3 "Squeaky Clean 2" sync / 1[56]:[^:]*:Silkie:7754:/
+54.8 "Squeaky Clean 3" sync / 1[56]:[^:]*:Silkie:(7755|7756):/
+61.9 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+64.0 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+69.0 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+73.0 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+80.6 "Carpet Beater" sync / 1[56]:[^:]*:Silkie:774F:/
+89.7 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+89.7 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+89.7 "Bracing Duster?" #sync / 1[56]:[^:]*:Silken Puff:7769:/
+89.7 "Chilling Duster?" #sync / 1[56]:[^:]*:Silken Puff:7768:/
+100.9 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+106.0 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+112.1 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+123.1 "Puff and Tumble" duration 0.7 #sync / 1[56]:[^:]*:Silken Puff:7771:/
+129.5 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+129.5 "Fizzling Duster x2" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+129.5 "Chilling Duster x2" #sync / 1[56]:[^:]*:Silken Puff:7768:/
+134.6 "Bracing Suds / Fizzling Suds" sync / 1[56]:[^:]*:Silkie:(7757|7759):/ 
+136.7 "--sync--" sync / 1[56]:[^:]*:Silkie:79FB:/
+141.7 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+145.7 "Bracing Duster / Fizzling Duster" sync / 1[56]:[^:]*:Silkie:(7764|7765):/
+146.0 "Soapsud Static?" #sync / 1[56]:[^:]*:Silkie:77ED:/
+160.1 "Total Wash" sync / 1[56]:[^:]*:Silkie:7750:/
+166.2 "Fresh Puff" sync / 1[56]:[^:]*:Silkie:7766:/
+172.3 "Eastern Ewers" sync / 1[56]:[^:]*:Silkie:776D:/
+178.6 "Rinse" sync / 1[56]:[^:]*:Eastern Ewer:776F:/ duration 7.3
+188.1 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+188.1 "Bracing Duster / Chilling Duster / Fizzling Duster" #sync / 1[56]:[^:]*:Silken Puff:(7769|7768|776A):/
+188.1 "Fizzling Duster" #sync / 1[56]:[^:]*:Silken Puff:776A:/
+195.2 "Chilling Suds" sync / 1[56]:[^:]*:Silkie:7758:/
+202.3 "Carpet Beater" sync / 1[56]:[^:]*:Silkie:774F:/
+204.4 "--sync--" sync / 1[56]:[^:]*:Eastern Ewer:79FB:/
+209.4 "Slippery Soap" sync / 1[56]:[^:]*:Silkie:775E:/
+213.4 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+225.9 "Bracing Suds" sync / 1[56]:[^:]*:Silkie:7757:/
+233.0 "Soaping Spree" sync / 1[56]:[^:]*:Silkie:7767:/
+233.0 "Chilling Duster?" #sync / 1[56]:[^:]*:Silken Puff:7768:/
+233.0 "Bracing Duster?" #sync / 1[56]:[^:]*:Silken Puff:7769:/
+233.0 "Fizzling Duster?" #sync / 1[56]:[^:]*:Silken Puff:776A:/
```

Comment:
```suggestion
233.0 "Bracing Duster / Chilling Duster / Fizzling Duster" #sync / 1[56]:[^:]*:Silken Puff:(7769|7768|776A):/
```

Reply:
Looks like I also missed a line here. The boss always casts Bracing Duster here as it always has Bracing Suds at this point when it does Soaping Spree.

Correctly resolving the Slippery Soap means there should be 6x Chilling Dusters going off here as well.